tools/src/wyvern/stdlib/support/AST.java

```diff
@@ -301,14 +308,76 @@ public IExpr parseExpression(String input, GenContext ctx) throws ParseException
         }
     }
 
+    public IExpr parseExpressionNoContext(String input) throws ParseException {
```

Comment:
Does the code in this function duplicate code elsewhere, and if so can we reuse it rather than duplicating?

Reply:
This method does call getNewAST and generateIL like parseExpression, but parseExpression calls generateIL with an empty list of TypedModuleSpec for dependencies. We've had to load them with the import statements so that there's no variable not found error when it's interpreted. Would it be better to move the bulk of this method into parseExpression and call that within this method?